src/ophys_etl/workflows/readme.md

```diff
@@ -2,83 +2,51 @@
 
 ## Installation
 
-If airflow not already installed:
+### Set up app DB
 
-`./scripts/install_airflow.sh`
+Use the connection details in section 1.2.1 of [this guide](http://confluence.corp.alleninstitute.org/pages/viewpage.action?pageId=60855687) to setup a db
 
-### Setup environment variables
-    
-Required environment variables:
-
-| Env var                           | Description                                     |
-|-----------------------------------|-------------------------------------------------|
-| AIRFLOW_HOME                      | Should point to root directory containing dags. |
-| OPHYS_WORKFLOW_APP_CONFIG_PATH    | Path to app config                              |
-
-### If from scratch:
-1. Using sqlite db
-
-    Initialize db using `airflow db init`. This populates the airflow database
-
-    Create users using `airflow users create`
-
-2. Using postgres
-
-   1. If For development use the connection details in section 1.2.1 of [this guide](http://confluence.corp.alleninstitute.org/pages/viewpage.action?pageId=60855687) to setup a dev db
-   > **_NOTE:_**  Use port 5412 for the dev postgres server, as the version at the default port 5432 does not work
-
-   2. Create a user using the instructions [here](https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#setting-up-a-postgresql-database)
-   3. Modify the variable `sql_alchemy_conn` in `airflow.cfg` found within `AIRFLOW_HOME`
-   4. Run `airflow db init`
-   5. Create users using `airflow users create`
-   
 Initialize ophys workflow db using `python -m ophys_etl.workflows.db.initialize_db`
 
-### start airflow
-
-`./scripts/setup.sh`
-
-## Testing
+### Launch app using docker compose
 
-A script that can be used for testing is `ophys_etl/workflows/scrips/run_pipeline_end_to_end_test.py`
-
-To run script from scratch:
-
-1. stop airflow services
+In same dir as `workflows/docker-compose-yaml`, run
+```bash
+BASE_DIR=<base directory to store logs in>
+AIRFLOW_WEBSERVER_HOST_PORT=<port to map webserver to on the host>
 
-    `pkill airflow`
+echo -e "AIRFLOW_UID=$(id -u)" >> .env
+echo -e BASE_DIR=$BASE_DIR >> .env
+echo -e AIRFLOW_WEBSERVER_HOST_PORT=$AIRFLOW_WEBSERVER_HOST_PORT >> .env
+```
 
-2. Reset airflow db
-    ```bash
-    airflow db reset
-    airflow db init
-    airflow users create ...
-    ```
+Place app_config_dev.yml in same directory as docker-compose.yaml
```

Comment:
Likely add mention of getting an example app config from s3://ophys-processing-airflow.alleninstitute.org/prod/ here as well.